eventhorizon_test.go

```diff
@@ -54,7 +54,7 @@ type TestEvent struct {
 }
 
 func (t *TestEvent) AggregateID() UUID     { return t.TestID }
-func (t *TestEvent) AggregateType() string { return "Test" }
+func (t *TestEvent) AggregateType() string { return "TestAggregate" }
```

Comment:
Does this trigger a test fail? It would be nice to add a test case that tests applying one event of the correct aggregate type and one of another type, checking that only the correctly typed event has been applied. Would it be possible for you to add that to the PR?